Makefile

```diff
@@ -7,6 +7,10 @@ HANDLER_IMAGE_NAME ?= kubernetes-nmstate-handler
 HANDLER_IMAGE_SUFFIX ?=
 HANDLER_IMAGE_FULL_NAME ?= $(IMAGE_REPO)/$(HANDLER_IMAGE_NAME)$(HANDLER_IMAGE_SUFFIX)
 HANDLER_IMAGE ?= $(IMAGE_REGISTRY)/$(HANDLER_IMAGE_FULL_NAME)
+OPERATOR_IMAGE_NAME ?= kubernetes-nmstate-operator
+OPERATOR_IMAGE_SUFFIX ?=
+OPERATOR_IMAGE_FULL_NAME ?= $(IMAGE_REPO)/$(OPERATOR_IMAGE_NAME)$(OPERATOR_IMAGE_SUFFIX)
+OPERATOR_IMAGE ?= $(IMAGE_REGISTRY)/$(OPERATOR_IMAGE_FULL_NAME)
 
 NAMESPACE ?= nmstate
```

Comment:
We have to split handler and operator manifests I suppose
HANDLER_NAMESPACE, OPERATOR_NAMESPACE ?